File: program/check-pangram-string/check_pangram_string.php
<?php
function isPangram($param) {
		
		$sentences = strtolower(trim($param));
		$letters = str_split("thequickbrownfoxjumpsoverthelazydog");

		foreach ($letters as $letter) {
			if (!strstr($sentences, $letter))
				return 'Not a Pangram sentence';
		}

		return 'Pangram sentence';
	}

echo isPangram('The quick brown fox jumps over the lazy dog');
?>

File: program/check-pangram-string/check-pangram-string.jl
function ispangram(input::AbstractString)
    input = lowercase(input)
    letters = Set('a':'z')
    for ch in input
        delete!(letters, ch)
    end
    isempty(letters)
end

if ispangram("The quick brown fox jumps over the lazy dog")
  println("Pangram")
else
 println("Not a Pangram")
end
